vault/provider.go

```diff
@@ -856,12 +871,19 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 	// Set the token to the generated child token
 	client.SetToken(childToken)
 
+	client = client.WithRequestCallbacks(api.RequireState(lastStates...)).WithResponseCallbacks()
```

Comment:
Why is this chained like so in there? If this is necessary, can you leave a comment around this line?